### ISSUE TITLE
Change classes

### ISSUE THREAD

**Issue by ChrisAngel447** [TimGeyssens/MCFly#1]
Hi Tim 

First off thank you for a great addon for your great addon UI-O-Matic.

I see that the styling of the fields are very straightforward and need to add a class to the textboxes and the text area. Is there any way I can change the style after McFly is installed?

Kind regards and thanks 

Chris

**Comment by TimGeyssens** [comment]
Yeah you can just add that to the views, they are located here ~/Views/Shared/EditorTemplates/...
